pywebdriver/views.py

```diff
@@ -61,7 +61,7 @@ def status():
 
 @app.route('/usb_devices.html', methods=['GET'])
 def usb_devices():
-    str_devices = commands.getoutput("lsusb").split('\n')
+    str_devices = subprocess.getoutput("lsusb").split('\n')
```

Comment:
```suggestion
    str_devices = subprocess.check_output("lsusb", universal_newlines=True).split('\n')
```

Reply:
@rousseldenis any idea about this suggestion ? Thanks

Reply:
@bealdav I think the suggestion is good.
@sbidoul As i read in docs https://docs.python.org/fr/3.7/library/subprocess.html
wouldn't be useful to catch CalledProcessError ?